app/models/merchant.rb

```diff
@@ -28,4 +28,9 @@ def distinct_cities
     item_orders.distinct.joins(:order).pluck(:city)
   end
 
+  def inactivate_items
+    items.each do |item|
+      item.update(active?: false)
+    end
+  end
```

Comment:
Good use of a model method

Reply:
I still think that we will be able to get this done using straight Active Record